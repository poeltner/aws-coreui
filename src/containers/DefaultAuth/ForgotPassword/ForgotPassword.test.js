import React from 'react';
import { shallow } from 'enzyme';
import DefaultForgotPassword from './ForgotPassword';

const acceptedStates = [
  'forgotPassword'
];

describe('forgotPassword', () => {
  describe('normal case', () => {
      test('render correctly with authState forgotPassword', () => {
          for (let i = 0; i < acceptedStates.length; i += 1){
              const wrapper = shallow(<DefaultForgotPassword/>);
              wrapper.setProps({
                  authState: acceptedStates[i],
              });
              expect(wrapper).toMatchSnapshot();
          }
      });
    });
});
