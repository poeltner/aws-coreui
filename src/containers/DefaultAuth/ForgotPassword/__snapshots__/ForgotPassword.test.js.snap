// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`forgotPassword normal case render correctly with authState forgotPassword 1`] = `ShallowWrapper {}`;
